### DISCUSSION TITLE
Relationship (if any) between the Drug Price Object fields and Pharmacies Object fields

### DISCUSSION THREAD

**Discussion by pdey6** [CMSgov/price-transparency-guide#75] [Q&A] [answered]
What is the relationship (if any) between the Drug Price Object fields ('**Pharmacies**' and '**NPI**' that are not required) and Pharmacies Object fields ('**pharmacy_ids**' and '**pharmacy_id_type**' which are required). How are they different? 
Will pricing be shown based on pharmacy_ids (use NPI?), pharmacy_id_type (use NPI type?) and NDC?

**Comment by shaselton-usds** [comment, marked as answer]
Wrapping up loose threads -- Rx file is no longer required at this point in time.
